packages/bpk-component-card-list/src/BpkCardList.tsx

```diff
@@ -16,31 +16,45 @@
  * limitations under the License.
  */
 
+import BpkBreakpoint, { BREAKPOINTS } from '../../bpk-component-breakpoint';
 import { BpkButtonV2 } from '../../bpk-component-button';
 import BpkSectionHeader from '../../bpk-component-section-header';
 import { cssModules } from '../../bpk-react-utils';
 
+import BpkCardListGridStack from './BpkCardListGridStack';
+import { LAYOUTS } from './common-types';
+
 import type CardListProps from './common-types';
 
 import STYLES from './BpkCardList.module.scss';
 
 const getClassName = cssModules(STYLES);
 
+const DEFAULT_ITEMS = 3;
+
 const BpkCardList = (props: CardListProps) => {
   const {
+    accessory,
     buttonHref,
     buttonText,
+    cardList,
     description,
+    expandText,
+    initiallyShownCards = DEFAULT_ITEMS,
+    layoutDesktop,
+    layoutMobile,
     onButtonClick,
     title,
   } = props;
 
-  const button = buttonText && (
-    <BpkButtonV2 onClick={onButtonClick} href={buttonHref}>{buttonText}</BpkButtonV2>
+  const button = buttonText && accessory !== 'button' && (
```

Comment:
likewise with this - a constant would be useful to avoid consumers misusing the component due to a typo